README.rst

```diff
@@ -70,3 +70,17 @@ Then in your pages:
     ${h.metatags(title="pagetitle", description="Page Description", image="http://url/myimage.png")}
 
 
+Storage
+=======
+
+``tgext.utils.storage`` is a tool for storing files into /public dir in separated folders.
+
+.. code-block:: python
+
+    from tgext.utils.storage import store
+
+    filename = store(ufile)  # ufile is an instance of cgi.FieldStorage
+
+file is stored inside /public/${uuid1} folder thus also accessible using internal tg file serving.
```

Comment:
Sorry I noticed this only now, but we surely want them to be uploaded in `/public/uploads/${uuid1}` or similar. As we don't want to make the developer angry at us for filling his public files directories with hundreds of entries.